release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -775,6 +775,11 @@ For more information about this more flexible SR-IOV network deployment during N
 ==== Update to HAProxy 2.6
 With this release, {product-title} updated to HAProxy 2.6.
 
+[id="ocp-4-14-max-log-length-sidecar"]
+==== Increased maximum log length while using a sidecar
+Previously, the maximum length of the syslog message in the Ingress Controller was 1024 bytes. Now, the maximum value can be increased.
```

Comment:
Two suggestions:
* The heading probably should mention that this new option is for Ingress Controllers.  
* The text should say "sidecar" rather than "syslog".  (Internally, sidecar logging actually does run a syslog instance inside the sidecar container, but that's an implementation detail, and I don't see syslog mentioned in other product documentation about sidecar logging.)
```suggestion
==== Support for configuring the maximum length with sidecar logging in the Ingress Controller
Previously, the maximum length of log messages when logging to a sidecar in the Ingress Controller was 1024 bytes. Now, the maximum value can be increased.
```